docs/content/en/docs/installation/_index.en.md

```diff
@@ -23,7 +23,7 @@ Helm](#without-helm).
 If you don't have a Kubernetes cluster, [here's a official guide to set one up](https://kubernetes.io/docs/setup/).
 
 {{% notice info %}}
-Fission requires Kubernetes 1.9 or higher
+Fission requires Kubernetes 1.6 or higher
```

Comment:
Hi @msshroff Sorry did you mean 1.16?

Reply:
yes, there is typo in fixing the typo :)